### ISSUE TITLE
I cannot understand the logic of the calculation of loss_det_occ, looking forward for your applying...

### ISSUE THREAD

**Issue by prinasi** [Robertwyq/PanoOcc#12]
In the pano_occ_head.py, about line 291, 
``` python
voxel_det_mask = torch.ones(1, det_occ.shape[1], det_occ.shape[2], det_occ.shape[3]).to(occ.device).long()
voxel_det = (voxel_semantics < 17) 
Idx = torch.where(voxel_det)
```
I think the voxel_det_mask should be initialized with zeros, and filled with ones to represent the ground truth labels of the forehead things later. Did I think wrong? Please give me some advice.
